server/src/server.ts

```diff
@@ -1,14 +1,39 @@
-import express from 'express';
-import cors from 'cors';
+import 'reflect-metadata';
+import dotenv from 'dotenv';
+import { buildSchema } from 'type-graphql';
+import mongoose from 'mongoose';
 
-const port = process.env.PORT || 5000;
-const app = express();
+import { ApolloServer } from 'apollo-server';
+import UserResolver from './resolvers/users-resolver';
+import ClassroomResolver from './resolvers/classrooms-resolver';
 
-// Middleware
-app.use(express.urlencoded({ extended: true }));
-app.use(express.json());
-app.use(cors());
+dotenv.config();
 
-app.listen(port, () => {
-  console.log(`Listening on port ${port}`);
-});
+export async function startServer(config: any): Promise<ApolloServer> {
```

Comment:
It is quite easy to define a config interface that will allow you to get rid of tha any here

Reply:
I'm not sure to understand ?

Reply:
I'd like to read config:Config instead of config:any
To do that you have to declare a Config interface describing the config object declared in server-config.ts 
also ServerConfig would be the right name for the interface